test/packaging/ansible/roles/docker-install/tasks/install-docker-SLES_old.yaml

```diff
@@ -0,0 +1,33 @@
+---
+
+# Use OpenSUSE 15.1 repo to install docker
```

Comment:
Just a question, why are we using the repo 15.1 if the versions are "12.3"
  and "12.4"?
Is it because it works and better to use a recent one?

Reply:
In the discontinued repos I could not find docker :( 

http://download.opensuse.org/distribution/12.3/repo/oss/

So I took the next one that has it and works 🤷 (that's why also it's adding the repo, installing, and removing the repo)

Reply:
Good, thanks.